prow/git/v2/interactor.go

```diff
@@ -25,6 +25,7 @@ import (
 	"strings"
 
 	"github.com/sirupsen/logrus"
+	"k8s.io/test-infra/prow/git"
```

Comment:
This import should not exist

Reply:
Well, it seems that we actually need it since this is a different package.